LaunchUpCorp/Speakers-corner
# [SPE-4] Create Sign Up UI Page

## Pull Request Title (Using conventional commit Standard)

### Type of Change

- [x ] New Feature

### Description

I created the sign-up page using MaterialUI and Tailwind. The design is not final but the validation for the form works. 


Fixes #(issue)

### Testing Instructions

- Go to the jjdate/spe-4-create-sign-up-ui-page branch
- Run npm install
- Run npm run start
- Add an email address and a password

Validation:  
- The email must be valid 
- The password should be at least eight characters long, have one capital letter and one number

### Screenshots (optional)

#### Desktop
![image](https://user-images.githubusercontent.com/37986323/236591910-ec2392bc-24da-40a7-abfd-4495721456b6.png)

#### Tablet
![image](https://user-images.githubusercontent.com/37986323/236591935-41456330-b717-4cdc-b289-7d89188f5029.png)

#### Mobile
![image](https://user-images.githubusercontent.com/37986323/236591954-84b68403-61dd-4bf3-bc3a-2d0680e849b7.png)

### Deployment Considerations (optional)
N/A

### Checklist (delete irrelevant items)

- [x ] I have added the necessary documentation.
- [ x] I have followed coding standards and best practices.
- [x ] My changes generate no new warnings or errors.
- [ x] I have manually tested my changes.
